src/components/inputs/notes.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div>
+        <textarea            
+            v-model="newNote"           
+            :id="$attrs.id"
+            :class="$attrs.class"
```

Comment:
default class to bootstrp  `form-control`

Reply:
done